src/js/components/Button/stories/Busy.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { Box, Button } from 'grommet';
+
+export const Busy = () => {
+  const [busy, setBusy] = useState();
+  const [success, setSuccess] = useState();
+
+  return (
+    <Box align="center" pad="medium">
+      <Button
+        primary
+        busy={busy}
+        success={success}
+        label="Button Busy"
+        onClick={() => {
+          setBusy(true);
+          setTimeout(() => {
+            setBusy(false);
+            setSuccess(true);
+          }, 2000);
+          setTimeout(() => {
+            setSuccess(false);
+          }, 4000);
+        }}
+      />
+    </Box>
+  );
+};
+
```

Comment:
Maybe skip chromatic for this, since it doesn't show anything unique statically?